README.md

```diff
@@ -1 +1,70 @@
-# math-magicians
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] You've done well setting up the project with CRA. That aside, in every project, in the general requirements section, there is a requirement for proper documentation of our projects. The requirement is quoted below:

  > - Make sure that you documented your work in a professional way.
  
  Having this in mind, please let's go back and properly update the README to reflect the information relevant to the project under development. This will make our work look professional.